tools/snp.sh

```diff
@@ -164,6 +172,63 @@ verify_snp_host() {
   fi
 }
 
+verify_if_host_is_snp_capable() {
```

Comment:
Would it make sense to just create a single function here and pass parameters to determine guest/host?

Reply:
As per this, I created a single function "verify_platform_snp_bit_status" with guest/host parameter to handle snp bit check